incubator/prometheus/values.yaml

```diff
@@ -1,10 +1,200 @@
-image: prom/prometheus
-imageTag: v1.1.0
-imageAlertManager: prom/alertmanager
-imageTagAlertManager: v0.4.2
-
-# Persist data to a persitent volume
-persistence:
-  enabled: true
-  storageClass: generic
-  size: 8Gi
+alertmanager:
+  ## Alertmanager service port
+  ## Default: 80
+  ##
+  # httpPort: 80
+
+  ## Alertmanager service port name
+  ## Default: 'http'
+  ##
+  # httpPortName: http
+
+  ## Alertmanager Docker image
+  ##
+  image: prom/alertmanager:v0.4.2
+
+  ingress:
+    ## Alertmanager Ingress annotations
+    ##
+    # annotations:
+    #   kubernetes.io/ingress.class: nginx
+    #   kubernetes.io/tls-acme: 'true'
+
+    ## If true, Alertmanager Ingress will be created
+    ##
+    enabled: false
+
+    ## Alertmanager Ingress hostnames
+    ## Must be provided if Ingress is enabled
+    ##
+    # hosts:
+    #   - hostName: alertmanager.domain.com
+
+    ## Alertmanager Ingress TLS configuration
+    ## Secrets must be manually created in the namespace
+    ##
+    # tls:
+    #   - secretName: prometheus-alerts-tls
+    #     hosts:
+    #       - alertmanager.domain.com
+
+  ## Alertmanager resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    # limits:
+    #   cpu: 10m
+    #   memory: 32Mi
+    requests:
+      cpu: 10m
+      memory: 32Mi
+
+  ## Alertmanager data storage path
+  ## Default: '/data'
+  ##
+  # storage_path: /data
+
+## Configmap-reload Docker image
+##
+configmap_reload_image: jimmidyson/configmap-reload:v0.1
+
+## Global imagePullPolicy
+## Default: 'Always' if image tag is 'latest', else 'IfNotPresent'
+## Ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+server:
+  ## Server Pod annotations:
+  ##
+  # annotations
+  #   iam.amazonaws.com/role: prometheus
+
+  ## Additional Server container arguments
+  ##
+  extra_args:
+    - --storage.local.max-chunks-to-persist=524288
+    - --storage.local.memory-chunks=1048576
+    - --storage.local.retention=360h0m0s
+    - --web.external-url=http://prometheus.domain.com
+
+  ## Server service port
+  ## Default: 80
+  ##
+  # httpPort: 80
+
+  ## Server service port name
+  ## Default: 'http'
+  ##
+  # httpPortName: http
+
+  ## Server Docker image
+  ##
+  image: prom/prometheus:v1.2.1
+
+  ingress:
+    ## Server Ingress annotations
+    ##
+    # annotations:
+    #   kubernetes.io/ingress.class: nginx
+    #   kubernetes.io/tls-acme: 'true'
+
+    ## If true, Server Ingress will be created
+    ##
+    enabled: false
```

Comment:
I would put this directly underneath the ingress scope. It's easy to miss mid way